addons/website_sale_slides/views/slide_channel_views.xml

```diff
@@ -23,6 +23,18 @@
         </field>
     </record>
 
+    <record id="slide_channel_view_tree_report" model="ir.ui.view">
+        <field name="name">slide.channel.report.view.list.inherit.website_sale_slides</field>
```

Comment:
Naming should match convention used for ids, aka model.view.view_type.suffixes :) "slide.channel.view.tree.report.inherit.sale_slides" for example